### PR TITLE
yuzu-{mainline,ea}: {517,1377} -> {546,1480}

### PR DESCRIPTION
**Pull request by IvarWithoutBones**
###### Motivation for this change

cc @aanderse :)

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113949` at f7159cd8 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/e2dd01d5d28639ba22d0f1951c964386">yuzu-ea</a></li></li>
    <li><a href="https://gist.github.com/5a3e763d97b14ba273669d9d6e8fa96c">yuzu-mainline</a></li></li>
  </ul>
</details>

File: pkgs/misc/emulators/yuzu/default.nix [hunk 1]
```diff
@@ -4,25 +4,25 @@ let
...
     pname = "yuzu-ea";
-    version = "1377";
+    version = "1480";
     branchName = branch;
     src = fetchFromGitHub {
       owner = "pineappleEA";
       repo = "pineapple-src";
       rev = "EA-${version}";
-      sha256 = "0jjddmcqbkns5iqjwqh51hpjviw5j12n49jwfq7xwrsns6vbpqkf";
+      sha256 = "09mlgflhvvjcffk9mczxfn972w87x90gys9gdp2w92xb7zm72v55";
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/yuzu/default.nix:25]:
```suggestion
      sha256 = "0flc5mckmnr9gj8f78nh9nys96inlkqk3rvpgbpl0mhcg6lmlb2g";
```

Comment (IvarWithoutBones) [reply pkgs/misc/emulators/yuzu/default.nix]: Not sure how that managed to build locally, but I've fixed it.

**Comment by IvarWithoutBones** [comment]
I've just updated this to include version info at runtime, which previously did not work. 

Thank you @JoshuaFern for [noticing this](https://github.com/NixOS/nixpkgs/pull/110810#pullrequestreview-578894981)! I had no idea as I do not have titlebars in my WM.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113949` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>yuzu-ea</li>
<li>yuzu-mainline</li>
</ul>
</details>

**Pull request merged**